getting_started/step_by_step/signals.rst

```diff
@@ -354,6 +471,20 @@ names between parentheses:
 
     signal health_changed(old_value, new_value)
 
+    var health = 10
+
+ .. code-tab:: csharp C#
+
+    using Godot;
+
+    public class CustomSignal : Node
+    {
+        [Signal]
+        public delegate void HealthChanged(int oldValue, int newValue);
+
+        private int Health = 10;
```

Comment:
The C# naming convention for fields is camelCase.

```suggestion
        private int health = 10;
```

In the engine we also prefix private fields with an underscore `_` but I'm not sure if that also applies to the documentation.

Reply:
Reverted back to the uppercase variant to match it with the current docs. As well as other variables, like Speed and AngularSpeed.